CHANGELOG.md

```diff
@@ -23,6 +23,10 @@
 - cli: handle missing files during metadata apply (close #5163) (#5170)
 - docs: add page on Relay schema (close #4912) (#5150)
 
+### Docs
+
+- docs: add hasura cloud to getting started section (close #5206) (#5208)
```

Comment:
We usually just have highlights and other improvements. 

Can you add this in `Next release` section

Reply:
@tirumaraiselvan done